flashcards/backend/comments/models.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+
+
+class Comment(models.Model):
+    """Model for representing user's comments in response to flashcards and decks
+
+    :param user: related user object
+    :type user: User
```

Comment:
:type user: CustomUser this will generate a link to CustomUser in our documentation.